tools/SyncDocs.ps1

```diff
@@ -0,0 +1,63 @@
+[CmdletBinding()]
```

Comment:
It will be great if you add some comment for the script here. Such as `Sync docs from Azure/azure-powershell to MicrosoftDocs`. Because we may will have other sync docs pipeline.